### PR TITLE
Api test for tags

### PR DESCRIPTION
**Pull request by Kaushik1216**
Pull Request for Issue # .

### CODE REVIEW

**Comment by laoneo** [comment]
Nice, did you not do the create tag test on purpose?

**Comment by Kaushik1216** [comment]
In other component test for create option their exist a category , but for tags their category not exist . That's why I don't write create test

**Comment by laoneo** [comment]
Do you need a category to create a tag?

**Comment by brianteeman** [comment]
> Do you need a category to create a tag?

tags doesnt use categories

**Comment by Kaushik1216** [comment]
> Do you need a category to create a tag?

no but on create option of other component instread of creating contact for contact component .A contact with category is created

**Comment by laoneo** [comment]
So what exactly blocks you from creating a tag then?

**Comment by Kaushik1216** [comment]
> So what exactly blocks you from creating a tag then

Done .  I was confused a little bit.

File: tests/System/integration/api/com_tags/Tags.cy.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      title: 'automated test tag',
+      alias: 'automated test tag',
+      note: '',
+      description: '',
+      published: 1,
+      parent_id: 1,
+      level: 1,
+      path: 'test-tag',
+      access: 1,
+      lft: 1,
```
Comment (laoneo) [inline tests/System/integration/api/com_tags/Tags.cy.js]: Does it also work without this line?

Comment (Kaushik1216) [reply tests/System/integration/api/com_tags/Tags.cy.js]: yes . should I remove this lines ?

Comment (laoneo) [reply tests/System/integration/api/com_tags/Tags.cy.js]: Yes please, remove all the lines which are not needed.

**Comment by Kaushik1216** [comment]
Without arrow function create tag is not working . What should I have to do to correct it ?

File: tests/System/integration/api/com_tags/Tags.cy.js (hunk 1 above)
Comment (Kaushik1216) [reply tests/System/integration/api/com_tags/Tags.cy.js]: done

**Comment by laoneo** [comment]
Please review the code again, the error is very obvious.

File: tests/System/integration/api/com_tags/Tags.cy.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  it('can deliver a list of tags', () => {
+    cy.db_createTag({ title: 'automated test tag' })
+      .then(() => cy.api_get('/tags'))
+      .then((response) => cy.wrap(response).its('body').its('data.0').its('attributes')
+        .its('title')
+        .should('include', 'automated test tag'));
+  });
+
+  it('can create a tag', () => {
+    const test = ()=>cy.api_post('/tags', {
```
Comment (laoneo) [inline tests/System/integration/api/com_tags/Tags.cy.js]: This looks wrong

Comment (Kaushik1216) [reply tests/System/integration/api/com_tags/Tags.cy.js]: How we can make it work without arrow function ? api commands fails when not call after arrow unction

Comment (laoneo) [reply tests/System/integration/api/com_tags/Tags.cy.js]: It fails because it was incorrect. I fixed it for now

**Comment by Kaushik1216** [comment]
Why you choose some selected field ie (with what basis) for create tag test?

**Comment by laoneo** [comment]
This are the minimum required fields to create a tag, all the others have a default value or can be empty/null.

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**